src/display/editor/editor.js

```diff
@@ -52,6 +52,22 @@ class AnnotationEditor {
 
   static _zIndex = 1;
 
+  static FakeEditor = class extends AnnotationEditor {
+    constructor(params) {
+      super(params);
+      this.annotationElementId = params.annotationElementId;
+      this.deleted = true;
+    }
+
+    serialize() {
+      return {
+        id: this.annotationElementId,
+        deleted: true,
+        pageIndex: this.pageIndex,
+      };
+    }
+  };
+
```

Comment:
Why is this defined "inline" here, since that feels strange/unnecessary?
I really think that we should simply format this as a normal `class FakeEditor extends AnnotationEditor { ... }` placed *after* [this line instead](https://github.com/mozilla/pdf.js/blob/9af50dc358fe4e4fa1ca93db1f5eb5560391f382/src/display/editor/editor.js#L613).